### PR TITLE
Enable GitHub Actions CI runs for pull requests (from forks)

### PR DESCRIPTION
**Pull request by osma**
## Reasons for creating this PR

I noticed that CI runs are not being triggered for pull requests from forked repositories, e.g. #1276

## Link to relevant issue(s), if any

n/a

## Description of the changes in this PR

Enable GitHub Actions CI jobs for the `pull_request` event type.

## Known problems or uncertainties in this PR

n/a

## Checklist

- [x] phpUnit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if not, explain why below)
- [x] The PR doesn't introduce unintended code changes (e.g. empty lines or useless reindentation)

no need for new tests

### CODE REVIEW

**Review by kinow — APPROVED** [review]
+1, I normally add `workflow_dispatch` too so I can trigger builds manually too, but not really necessary here I guess. Thanks!

**Comment by kinow** [comment]
And feel free to rebase my PR if you'd like to test it (not sure if that'd be picked by GH Actions, but just in case) :wave:

**Comment by osma** [comment]
>  +1, I normally add workflow_dispatch too so I can trigger builds manually too, but not really necessary here I guess. Thanks!

Great idea, I just did that in another commit!

> And feel free to rebase my PR if you'd like to test it (not sure if that'd be picked by GH Actions, but just in case) wave

Maybe you could rebase your PR after this one has been merged to `master`? I will do it soon, just waiting for the CI jobs to complete.

**Pull request merged**
